config/locales/views/grader_permissions/en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  grader_permissions:
+    list: "Grader Permissions"
+    manage_assessments: "Manage assessments"
+    manage_submissions: "Manage submissions"
+    run_tests: "Run tests"
+    tooltip1: "Create and manage all the assessments which includes annotations, automated testing, exam templates."
+    tooltip2: "Collect and manage submissions, and release/unrelease the grades."
```

Comment:
Good idea with the tooltips!
Please name them something more descriptive than "tooltip1" and "tooltip2". Also please add a 3rd tooltip for the "Run tests" option.